pkgs/by-name/yo/yosys/package.nix

```diff
@@ -29,7 +29,7 @@
   yosys-ghdl,
   yosys-symbiflow,
   nix-update-script,
-  enablePython ? true, # enable python binding
+  enablePython ? false, # enable python binding
```

Comment:
Can you please provide an explanation why you're making this change?

Reply:
I can't remember exactly... Maybe just to test if this was helping with building the package.
I'll revert it.